cmd2/cmd2.py

```diff
@@ -1093,8 +1093,8 @@ def complete_users():
             self.allow_appended_space = False
             self.allow_closing_quote = False
 
-        # Sort the matches before any trailing slashes are added
-        matches = utils.alphabetical_sort(matches)
+        # Sort the matches alphabetically before any trailing slashes are added
+        matches.sort(key=utils.norm_fold)
```

Comment:
Nice little performance improvement to prevent making an extra copy

Reply:
Thanks!